run-cnf-suites.sh

```diff
@@ -36,6 +36,9 @@ done
 GINKGO_ARGS="-ginkgo.v -junit . -report $OUTPUT_LOC -claimloc $OUTPUT_LOC"
 FOCUS=""
 
+# specify Junit report file name.
+GINKGO_ARGS2="-ginkgo.reportFile $OUTPUT_LOC/"cnf-certification-tests_junit.xml""
```

Comment:
Please add this to the existing `GINKGO_ARGS` declaration rather than specify a new one.